jetty/src/main/java/com/opentable/logging/jetty/JsonRequestLog.java

```diff
@@ -81,7 +81,7 @@ public void log(final Request request, final Response response)
         }
 
         final HttpV1 payload = createEvent(request, response);
```

Comment:
Is `createEvent` a function the programmer overrides?  If so, consider adding a `@NonNull` annotation.  That'll get included if you override, right?  That might help detect inappropriately returning `null` a little earlier, avoiding the runtime error.